### DISCUSSION TITLE
Star Vijay India

### DISCUSSION THREAD

**Discussion by Angu0083** [iptv-org/community#738] [Q&A]
Channel Name: Star Vijay
Channel ID: StarVijay.in
Country: India
Language: Tamil

**Comment by BellezaEmporium** [comment]
@freearhey https://en.wikipedia.org/wiki/Star_Vijay => Asianet Star Communications : https://en.wikipedia.org/wiki/Asianet_Star_Communications

**Comment by freearhey** [comment]
@LaneSh4d0w I used lyngsat as a source: https://www.lyngsat.com/tvchannels/hk/Star-Vijay-India.html

**Comment by BellezaEmporium** [comment]
That's maybe because of a licencing agreement, i don't know.

To push things even further, we can even say it's US, Star being a Walt Disney subsidiary. (Asianet and Disney made an alliance, becoming Asianet Star Communications and they have control all over the Asianet channels.)

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19737
